llvm/tools/llvm-exegesis/lib/MCInstrDescView.h

```diff
@@ -160,12 +166,17 @@ struct Instruction {
   const BitVector &ImplUseRegs; // The set of aliased implicit use registers.
   const BitVector &AllDefRegs;  // The set of all aliased def registers.
   const BitVector &AllUseRegs;  // The set of all aliased use registers.
+  const BitVector &MemoryRegs;  // The set of all aliased memory use registers.
+  const BitVector
```

Comment:
This formatting looks odd... maybe we could just put the comment above the line so that clang-format won't do anything funny?

Reply:
Addressed